config/initializers/flipper.rb

```diff
@@ -1,6 +1,15 @@
 # frozen_string_literal: true
 
 require "flipper/adapters/redis"
+
+# default flipper configuration
+default_flipper_features = {
+  'recaptcha': false,
+  'forum': false,
+  'post_comments': false,
+  'project_comments': true
```

Comment:
what is the difference between these two ?

Reply:
tbh, I don't know, I just put it because https://circuitverse.org/flipper has both of them. Only `project_comments` is used.